addons/web/static/tests/helpers/test_utils_mock.js

```diff
@@ -290,6 +290,9 @@ function intercept(widget, eventName, fn, propagate) {
  */
 function removeSrcAttribute(el, rpc) {
     var nodes;
+    if (el.nodeName === "#comment") {
```

Comment:
el.nodeType === 8 (just like in qweb Owl or legacy qweb)

Reply:
So line below already fixes this?
```js
el = el.nodeType === 8 ? el.nextSibling : el;
```
If so, ok to remove. We introduced this fix some months ago.

Reply:
confirmed, this change can be removed, the line below has been added very very recently with dc260c324170a90fa737cd21dd528e39d658d9b6

Reply:
By removing this, I have cannot read `nodeName` of `undefined`. Possibility case of el.nextSibling being also a `#comment`?